### PR TITLE
f-cookie-banner@v4.0.0 – Moving component dependencies to peerDeps

### PR DESCRIPTION
**Pull request by kevinrodrigues**
### Added
- **Breaking Change**: Added `f-link` dependency to peer dependencies. Now `f-link` should be included as a dependency of the consuming component or application.
- **Breaking Change**: Added `f-button` dependency to peer dependencies. Now `f-button` should be included as a dependency of the consuming component or application.
- **Breaking Change**: Added `f-mega-modal` dependency to peer dependencies. Now `f-mega-modal` should be included as a dependency of the consuming component or application.

### Removed
- **Breaking Change**: Removed `f-link` styles import from the component. Make sure to import `f-link` styles in your application.
- **Breaking Change**: Removed `f-button` styles import from the component. Make sure to import `f-button` styles in your application.
- **Breaking Change**: Removed `f-mega-modal` styles import from the component. Make sure to import `f-mega-modal` styles in your application.

- [x] README and/or UI Documentation has been [created|updated]
- [ ] Unit tests have been [created|updated]
- [ ] This code has been checked with regard to [our accessibility standards](http://fozzie.just-eat.com/documentation/general/accessibility/checklist)

## Browsers Tested

- [ ] Chrome (latest)
- [ ] Internet Explorer 11
- [ ] Mobile (Please list device/browser – Ideally one iPhone model and one Android)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: packages/components/organisms/f-cookie-banner/package.json [hunk 1]
```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
+    "@justeat/f-button": "3.0.2",
+    "@justeat/f-link": "2.0.0",
+    "@justeat/f-mega-modal": "3.0.2"
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/package.json]: These should all be set as ranges to allow different versions to be used across consuming apps or components.

Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/package.json]: Good spot. 👍

Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/package.json]: Fixed.

File: packages/components/organisms/f-cookie-banner/package.json [hunk 2]
```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
+    "@justeat/f-button": "3.x.x",
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/package.json]: I think the syntax is `3.x` (if you want to limit it to version 3 and no higher) though I haven't tested to see what happens with the extra `x`.

File: packages/components/organisms/f-cookie-banner/package.json [hunk 3]
```diff
@@ -47,7 +47,10 @@
   },
   "peerDependencies": {
     "@justeat/browserslist-config-fozzie": ">=1.1.1",
-    "js-cookie": "2.2.1"
+    "js-cookie": "2.2.1",
```
Comment (DamianMullins) [inline packages/components/organisms/f-cookie-banner/package.json]: If you are able to could you also update this one to be a range, please? i.e. `>=2.2.1` like the browserslist one above.

File: packages/components/organisms/f-cookie-banner/package.json (hunk 2 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/package.json]: Should work but makes sense to do the `3.x` as that's the same as `3.x.x` syntax i believe. Updated it now.

File: packages/components/organisms/f-cookie-banner/package.json (hunk 3 above)
Comment (kevinrodrigues) [reply packages/components/organisms/f-cookie-banner/package.json]: fixed.

**Review by DamianMullins — DISMISSED** [review]

**Comment by ashleynolan** [comment]
Looks like the build is failing atm – not quite sure why from a quick look, seems it's looking for an element that doesn't exist?

**Comment by kevinrodrigues** [comment]
> Looks like the build is failing atm – not quite sure why from a quick look, seems it's looking for an element that doesn't exist?

Weird. Will take a look Monday. 👍

**Review by simonsandersje — DISMISSED** [review]

**Comment by ashleynolan** [comment]
@kevinrodrigues Looks like some of the Visual tests are failing and need the f-button styles?

**Review by DamianMullins — CHANGES_REQUESTED** [review]
@kevinrodrigues Looks like you lost your `peerDependencies` changes with that last commit?

**Comment by kevinrodrigues** [comment]
> @kevinrodrigues Looks like you lost your `peerDependencies` changes with that last commit?

Thanks @DamianMullins, my bad whilst resolving conflicts. Should be fixed now.

**Comment by kevinrodrigues** [comment]
> @kevinrodrigues Looks like some of the Visual tests are failing and need the f-button styles?
I think this should be fixed now. 👍

**Review by DamianMullins — DISMISSED** [review]

**Review by DamianMullins — DISMISSED** [review]

**Comment by DamianMullins** [comment]
@kevinrodrigues Is this PR still relevant?

**Pull request closed without merging**